odxtools/__init__.py

```diff
@@ -67,6 +67,7 @@
 from .loadfile import load_files  # noqa: F401
 from .loadfile import load_odx_d_file as load_odx_d_file  # noqa: F401
 from .loadfile import load_pdx_file as load_pdx_file  # noqa: F401
+from .loadfile import load_xml_file as load_xml_file  # noqa: F401
```

Comment:
For end-user, they only need `load_file` which does not seem to be already exposed, no need for `load_pdx_file`, `load_pdx_file` and `load_odx_d_file`